JetNews/.github/workflows/ci.yaml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  ci:
+    name: Build + Test
+    runs-on: macOS-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+#      TODO: Cache dependencies using actions/cache
+
+#      TODO: Run Ktlint
+#      - name: Ktlint
+#        run: ./gradlew ktlint
+
+      - name: Build project
+        run: ./gradlew assemble --stacktrace
+
+      - name: Run instrumentation tests
+        uses: reactivecircus/android-emulator-runner@v2
+        with:
+          api-level: 21
```

Comment:
Any reason for 21? 28/29 has a lot of performance improvements for the emulator iirc

Reply:
You use a matrix and run on various API levels too.

Reply:
Good idea! Adding the matrix with API 21 and 29! Thanks :) 

I generally like to cover the UI tests with the min API supported